### PR TITLE
Added scala-java8-compat as a dependency to streams

### PR DESCRIPTION
**Pull request by drewhk**

### CODE REVIEW

**Comment by ktoso** [comment]
LGTM

**Comment by patriknw** [comment]
so we are not using the imperative present tense style any more? ;-)
it's also missing issue number

**Comment by drewhk** [comment]
There is no issue number for this one. I can create a ticket if preferred, I just wanted to have this as a separate PR from all the other tickets that basically need this done to avoid merge conflicts.

**Comment by patriknw** [comment]
ok, but you could link it to the issue about CompletionStage

**Comment by drewhk** [comment]
Ok, then I amend this

File: project/Dependencies.scala [hunk 1]
```diff
@@ -54,7 +54,10 @@ object Dependencies {
     val jackson     = "com.fasterxml.jackson.core"    % "jackson-databind"             % "2.4.3"       // ApacheV2
 
     // For akka-http-testkit-java
-     val junit       = "junit"                         % "junit"                        % "4.11"        // Common Public License 1.0
+    val junit       = "junit"                         % "junit"                        % "4.11"        // Common Public License 1.0
+
+    // For Java 8 Conversions
+    val java8Compat = "org.scala-lang.modules"       %% "scala-java8-compat"           % "0.7.0"
```
Comment (patriknw) [inline project/Dependencies.scala]: add licence comment also

**Comment by patriknw** [comment]
LGTM, aside from nitpicking

**Pull request closed without merging**